content/en/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -189,7 +190,12 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
     To demonstrate this, let's spin up a simple Deployment and Pods in the `development` namespace.
 
     ```shell
+<<<<<<< HEAD
     kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development --replicas=2
+=======
+    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname -n=development
+    kubectl scale deployment snowflake --replicas=2 -n=development
+>>>>>>> upstream/master
```

Comment:
This looks a lot like a not-yet-resolved conflict that got committed in error.

Reply:
/lgtm cancel

Reply:
@savitharaghunathan you can:
- edit that to fix it
- `git add content/en/docs/tasks/administer-cluster/namespaces.md `
- `git commit --amend`
- `git push origin:merged-master-dev-1.19 --force-with-lease` (assuming that the remote for your fork is called “origin”)

Reply:
Good catch @sftim. Just checking, this is a fix to the master branch?

Reply:
The merge residue is in this branch.

Reply:
@sftim I have fixed this and force pushed the branch. Thank you for the commands :)